docs/reference/settings/notification-settings.asciidoc

```diff
@@ -35,8 +35,10 @@ required. For more information, see
 {xpack-ref}/encrypting-data.html[Encrypting sensitive data in {watcher}].
 
 `xpack.watcher.history.cleaner_service.enabled`::
-added[6.3.0,Default changed to `true`.]
-deprecated[7.0.0,Watcher history indices are now managed by the `watch-history-ilm-policy` ILM policy]
+// {empty} attributes are included so both macros render as intended.
```

Comment:
I think I'd prefer to use `ifdef::asciidoctor[]` blocks to handle the different way to write this. I think this way works but we only need it if we're going to support both Asciidoctor and AsciiDoc with the same source exactly. I think having `ifdef::asciidoctor[]` will let us search for it after the migration and we can remove all of the usages. Though we could also remove all the usages of `{empty}`. Either way is fine, but I think `ifdef::asciidoctor[]` is a little more consistent.

Reply:
The `{empty}` usage is needed to get Asciidoctor to render correctly, not AsciiDoc. I'm not sure we can remove it after migration, even if we implement `ifdef::asciidoctor[]`.

It seems Asciidoctor won't render two consecutive inline macros without any text between. The `{empty}` attribute is a hacky way around that.

I'm happy to implement `ifdef::asciidoctor[]` if we find a better way of rendering the two inline macros in Asciidoctor. My research hasn't turned up any alternative solutions though.

Reply:
Does this work for Ascidoctor?
```
added:[6.3.0,Default changed to `true`.]
deprecated:[7.0.0,Watcher history indices ...]
```

Reply:
Unfortunately not.

**Input**
```
added[6.3.0,Default changed to `true`.]
deprecated[7.0.0,Watcher history indices are now managed by the `watch-history-ilm-policy` ILM policy] 
```
**Asciidoctor output**
<img width="776" alt="Screen Shot 2019-05-01 at 1 50 34 PM" src="https://user-images.githubusercontent.com/40268737/57032463-72a5e380-6c18-11e9-85ba-8591222fc3c2.png">

Reply:
With the single `:`s between the name and the `[`. That is asciidoctor's native syntax. Nothing between the name and the `[` is a thing we twist into place based on heuristics that don't work here.

Reply:
:doh: That did the trick. Fixed it up with f31252b. Thanks again for your near-infinite patience and help, @nik9000.

Reply:
Pushed f7ba3b0 to implement `ifdef::asciidoctor[]`. Thanks again, Nik!